Documentation/operations/system_requirements.rst

```diff
@@ -84,6 +85,9 @@ RancherOS_                 >= 1.5.5
 .. [#centos_foot] CentOS 7 requires a third-party kernel provided by `ElRepo <http://elrepo.org/tiki/tiki-index.php>`_
     whereas CentOS 8 ships with a supported kernel.
 
+.. [#raspbian_foot] Raspbian requires a custom-built kernel because some functionalities are not enabled in the default
```

Comment:
Could you elaborate more about which features are missing? Otherwise, readers can't make sure which features they should enable for their custom kernel.

Reply:
Not sure if it's worth to have the list, it might change in the future. Assuming they're all in current requirements, I'd rather have a link pointing to the [required options](https://docs.cilium.io/en/latest/operations/system_requirements/#base-requirements) instead (``:ref:`admin_kernel_version` ``).

Reply:
Yep, that makes sense.

@jackap Please go with this direction.